diff -Naur linux-4.2.3-orig/include/linux/module.h linux-4.2.3/include/linux/module.h
--- linux-4.2.3-orig/include/linux/module.h	2015-10-03 13:52:18.000000000 +0200
+++ linux-4.2.3/include/linux/module.h	2016-01-11 17:58:45.299199829 +0100
@@ -487,6 +487,7 @@
 #endif
 
 extern struct mutex module_mutex;
+extern struct list_head modules;
 
 /* FIXME: It'd be nice to isolate modules during init, too, so they
    aren't used before they (may) fail.  But presently too much code
diff -Naur linux-4.2.3-orig/kernel/module.c linux-4.2.3/kernel/module.c
--- linux-4.2.3-orig/kernel/module.c	2015-10-03 13:52:18.000000000 +0200
+++ linux-4.2.3/kernel/module.c	2016-01-11 17:59:21.694659166 +0100
@@ -100,7 +100,8 @@
  * (delete and add uses RCU list operations). */
 DEFINE_MUTEX(module_mutex);
 EXPORT_SYMBOL_GPL(module_mutex);
-static LIST_HEAD(modules);
+LIST_HEAD(modules);
+EXPORT_SYMBOL_GPL(modules);
 
 #ifdef CONFIG_MODULES_TREE_LOOKUP
 
